src/ajitSJune20/SumOfNumInString.java

```diff
@@ -0,0 +1,28 @@
+package ajitSJune20;
+
+public class SumOfNumInString {
+
+	public static int getSumOfAllDigits(String input) {
+		int sum = 0;
+		String temp = "";
+		for (int index = 0; index < input.length(); index++) {
+			char ch = input.charAt(index);
+			if (Character.isDigit(ch))
+				temp += ch;
+			else {
+				if (temp.length() != 0) {
+					sum += Integer.parseInt(temp);
+					temp = "0";
```

Comment:
If you are reinializing temp with "0".. that means your length will never gonna be zero again, so there is no point for the if statement above to check the length

Reply:
Oh got it for the first time it will need that..

Reply:
Thanks.